docs/source/bentocloud/how-tos/autoscaling.rst

```diff
@@ -52,7 +45,19 @@ Use the ``@bentoml.service`` decorator to set concurrency:
 
     If ``concurrency`` is not set, the Service will only be autoscaled based on CPU utilization, which may not be optimal for your Service.
 
-To determine the optimal value for ``concurrency``, we recommend conducting a stress test on your service using a load generation tool such as `Locust <https://locust.io/>`_ either locally or on BentoCloud. The purpose of the stress test is to identify the maximum number of concurrent requests your service can manage. After identifying this maximum, set the concurrency parameter to a value slightly below this threshold ensuring that the service has adequate headroom to handle traffic fluctuations.
+Setting this parameter means the Service will be automatically scaled on BentoCloud when the concurrent requests per replica exceeds the specified concurrency threshold.
+
+For instance, consider a scenario where ``concurrency`` is set to 32 and the Service is currently operating with 2 replicas. If the Service receives 100 concurrent requests, BentoCloud will automatically scale up to 4 replicas to effectively manage the increased traffic. Conversely, if the number of concurrent requests decreases to below 32, BentoCloud will intelligently scale down to 1 replica to optimize resource utilization.
```

Comment:
> For instance, consider a scenario where ``concurrency`` is set to 32 and the Service is currently operating with 2 replicas
Shall we make it more clear ? it have ~ 64 concurrent requests . it already reached a stable state as a premise

Reply:
Updated the example here.